app/models/miq_widget_set.rb

```diff
@@ -45,13 +45,15 @@ def add_to_dashboard_order
   def delete_from_dashboard_order
     return unless group_id
```

Comment:
if `group_id` is no longer used here, should this be changed to `return unless group`?